data_diff/diff_tables.py

```diff
@@ -100,17 +101,33 @@ def __iter__(self):
             self.result_list.append(i)
             yield i
 
-    def _get_stats(self) -> DiffStats:
+    def _get_stats(self, is_dbt:bool = False) -> DiffStats:
```

Comment:
@dlawin can I use the same approach here--defining `is_dbt` as an input to `_get_stats`, which is called by `get_stats_string`?

Reply:
yep